demo/Scripts/FMOD.gd

```diff
@@ -196,4 +196,7 @@ func sound_get_pitch(uuid):
 	
 func sound_set_pitch(uuid, pitch):
 	return FMOD.sound_set_pitch(uuid, pitch)
+
+func set_sound_3d_settings(dopplerScale, distanceFactor, rollOffScale):
```

Comment:
Use the system namespace in front of the method name. Makes things easier for auto-completion in the script editor.
```suggestion
func system_set_sound_3d_settings(dopplerScale, distanceFactor, rollOffScale):
```